apps/rig_auth/lib/rig_auth/jwt/TokenUtils.ex

```diff
@@ -0,0 +1,52 @@
+defmodule RigAuth.Jwt.TokenUtils do
+  use Joken.Config, default_signer: nil # no default signer. Do not set as that will break Jwks retrieval
+
+  use Rig.Config, [:secret_key, :rotation_flag, :jwks_endpoint]
+
+  @conf_var config()
```

Comment:
by doing this you effectively "freeze" the configuration into the compiled module - as long as the module is not recompiled, the configuration cannot be changed. Considering that Confex specifically allows to _reload_ configuration at runtime (e.g., by changing the environment variables with the default Confex adapter), I think it'd be better not call `config()` where you need it instead.